data/zilliqa/assets/zil18prrzet4cwf96nvk50le04pa73du4cyvklltdv/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "ART Line",
+  "symbol": "ALE",
+  "links": {
```

Comment:
We do not accept personal socials that are loosely connected to token except in rare cases. Since your socials are empty too, they just seem like an attempt to inflate the score. You can remove all the links object and I will merge the PR

Reply:
data/zilliqa/assets/zil18prrzet4cwf96nvk50le04pa73du4cyvklltdv/meta.json

Reply:
data/zilliqa/assets/zil18prrzet4cwf96nvk50le04pa73du4cyvklltdv/meta.json

Reply:
data/zilliqa/assets/zil18prrzet4cwf96nvk50le04pa73du4cyvklltdv/logo.svg